lib/theme_check/checks.rb

```diff
@@ -8,5 +8,18 @@ def call(method, *args)
         end
       end
     end
+
+    # Return a collection of checks except for disabled checks represented
+    # as a DisabledChecks instance — or `nil`, if all checks are meant to be disabled.
+    # Still returns checks marked as `always_enabled!`
+    def except_disabled(disabled_checks = nil)
```

Comment:
It's not clear to me what the method arg does. Maybe it would be simpler if it was split in two methods: one the same as `disabled_check=nil`, and the other when present.

Reply:
I tried clearing this up by splitting out the methods as suggested. Still feels a bit off to me, so please let me know if you have suggestions!